### PR TITLE
Add a note about the visibility expansion of DomainMapping

### PR DESCRIPTION
**Pull request by ZhiminXiang**
Add a note to indicate that private Knative Serivce could be accessed from public internet when using DomainMapping.
See discussion knative-sandbox/net-istio#566

### CODE REVIEW

File: docs/serving/creating-domain-mappings.md [hunk 1]
```diff
@@ -43,6 +43,9 @@ Service. For example in the example yaml below, the "example.org" Domain
 Mapping maps only "example.org" and not "www.example.org". You can create
 multiple Domain Mappings to map multiple domains and subdomains.
 
+**Note:** If you create a Domain Mapping to map to a [private Knative Service](https://github.com/knative/docs/blob/mkdocs/docs/developer/serving/services/private-services.md), the private Knative Service will be accessible from public internet with the
```
Comment (csantanapr) [inline docs/serving/creating-domain-mappings.md:46]: Do not use full url to md file, instead put a relative link `services/private-services.md`

Comment (csantanapr) [inline docs/serving/creating-domain-mappings.md:46]: Do use `**Note**` now we have mkdocs and there is a new way using [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/)
Use something like
```
!!! Note
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla et euismod
    nulla. Curabitur feugiat, tortor non consequat finibus, justo purus auctor
    massa, nec semper lorem quam in mass.
```

**Review by csantanapr — CHANGES_REQUESTED** [review]
You also need to rebase to solve the conflicts

File: docs/serving/creating-domain-mappings.md (hunk 1 above)
Comment (ZhiminXiang) [reply docs/serving/creating-domain-mappings.md]: Done

Comment (ZhiminXiang) [reply docs/serving/creating-domain-mappings.md]: Done

**Comment by ZhiminXiang** [comment]
ok. looks like there was a big refactoring of the original file. I am gonna close this PR and add my stuffs to the new file.

**Pull request closed without merging**